packaging/hudi-flink-bundle/pom.xml

```diff
@@ -501,8 +501,7 @@
     <dependency>
       <groupId>org.apache.avro</groupId>
       <artifactId>avro</artifactId>
-      <!-- Override the version to be same with Flink avro -->
-      <version>1.10.0</version>
+      <version>${avro.version}</version>
```

Comment:
Make sure that there are no compatibility issues in flink side.

Reply:
Yes,  we've tested with Flink streamer loading data from Kafka datasource in Hudi format. And it works fine

Reply:
👍

Reply:
> Yes, we've tested with Flink streamer loading data from Kafka datasource in Hudi format. And it works fine
Hi, @CTTY  I met a java.lang.ClassNotFoundException when using the latest master code.   Class org.apache.avro.LogicalTypes$LocalTimestampMillis  seems to only appear in avro 1.10 version. Please help to confirm this problem, correct me if I am wrong . 
```
Caused by: java.lang.ClassNotFoundException: org.apache.hudi.org.apache.avro.LogicalTypes$LocalTimestampMillis
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382) ~[?:1.8.0_202]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_202]
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349) ~[?:1.8.0_202]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_202]
        at org.apache.hudi.table.HoodieTableFactory.inferAvroSchema(HoodieTableFactory.java:346) ~[hudi-flink1.14-bundle-0.13.0-SNAPSHOT.jar:0.13.0-SNAPSHOT]
```

Reply:
Yes, you can revert this avro version change to unblock yourself for now

Reply:
Can we fire a fix here?